tests/actors/rust/Cargo.toml

```diff
@@ -20,4 +21,5 @@ serde = { version = "1", default-features = false }
 serde_json = { version = "1", default-features = false }
 tokio = { version = "1", default-features = false }
 wasmcloud-actor = { path = "../../../crates/actor", default-features = false }
-wit-bindgen = { version = "0.12", default-features = false }
+wasmcloud-compat = { versoin = "*", path = "../../../crates/compat" }
```

Comment:
```suggestion
wasmcloud-compat = { version = "*", path = "../../../crates/compat" }
```

Reply:
`version` is unnecessary here (and `wasmcloud-compat` won't be either once this draft is done), removing